src/ingestion/Logary.Ingestion.HTTP/AssemblyInfo.fs

```diff
@@ -2,17 +2,17 @@
 namespace System
 open System.Reflection
 
-[<assembly: AssemblyTitleAttribute("Logary.Ingestion.HTTP")>]
-[<assembly: AssemblyProductAttribute("Logary.Ingestion.HTTP")>]
+[<assembly: AssemblyTitleAttribute("Logary.Ingestion.Http")>]
```

Comment:
Do you need to `rm -rf Logary.Ingestion.Http` and then `git checkout -- Logary.Ingestion.HTTP` to get the folder name correct, perhaps?

Reply:
it works, finally know why this happened